yt/frontends/ramses/tests/test_outputs.py

```diff
@@ -296,3 +296,16 @@ def check_unit(array, unit):
         check_unit(ds.r['ramses', 'Bar'], 'dimensionless')
     finally:
         ytcfg.remove_section('ramses-hydro')
+
+@requires_file(output_00080)
```

Comment:
nice, it's awesome we don't even need to upload a new data file. it's great when we can unlock more features for data we support already :)

Reply:
Sure! I was very pleased by the small quantity of code I had to write to add support for this ;)